app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -11,5 +11,12 @@ def permissions
         }
       end
     end
+
+    api :GET, '/admin/states', 'Get current user\' info'
+    def info
+      success do
+        current_user_info
```

Comment:
Also, if you have a simple value to render using `success`, simply use `success current_user_info`. No need for the block.